crates/youki/src/commands/features.rs

```diff
@@ -1,8 +1,171 @@
 //! Contains Functionality of `features` container command
 use anyhow::Result;
+use libcontainer::oci_spec::runtime::{
+    version, ApparmorBuilder, CgroupBuilder, FeaturesBuilder, IDMapBuilder, IntelRdtBuilder,
+    LinuxFeatureBuilder, LinuxNamespaceType, MountExtensionsBuilder, SelinuxBuilder,
+};
 use liboci_cli::Features;
 
+// Function to query and return capabilities
+fn query_caps() -> Result<Vec<String>> {
+    Ok(caps::all().iter().map(|cap| format!("{:?}", cap)).collect())
+}
+
+// Function to query and return namespaces
+fn query_supported_namespaces() -> Result<Vec<LinuxNamespaceType>> {
+    Ok(vec![
+        LinuxNamespaceType::Pid,
+        LinuxNamespaceType::Network,
+        LinuxNamespaceType::Uts,
+        LinuxNamespaceType::Ipc,
+        LinuxNamespaceType::Mount,
+        LinuxNamespaceType::User,
+        LinuxNamespaceType::Cgroup,
+        LinuxNamespaceType::Time,
+    ])
+}
+
+// Return a list of known hooks supported by youki
+fn known_hooks() -> Vec<String> {
+    [
+        "prestart",
+        "createRuntime",
+        "createContainer",
+        "startContainer",
+        "poststart",
+        "poststop",
+    ]
+    .iter()
+    .map(|s| s.to_string())
+    .collect()
+}
+
+// Return a list of known mount options supported by youki
+fn known_mount_options() -> Vec<String> {
```

Comment:
This may be slightly beyond the scope of this PR, but I'm considering methods to ensure this list is regularly updated. WDYT?

Reply:
OK, I'll try.

Reply:
Hey, unless either of you have done implementation of the updation, I think we should go ahead with hard-coded parts in this PR and merge once reviewed, and the dynamic values can be done in https://github.com/containers/youki/pull/2937 or so. WDYT?

Reply:
This function will be replaced with `MountOption::known_options` once https://github.com/youki-dev/youki/pull/2937 gets merged.